src/sentry/static/sentry/app/views/settings/projectGeneralSettings.jsx

```diff
@@ -268,12 +268,14 @@ class ProjectGeneralSettings extends AsyncView {
           <JsonForm
             {...jsonFormProps}
             title={t('Data Privacy')}
+            features={new Set(organization.features)}
```

Comment:
I think for these forms we just keep `organization.features` as an array and check using `Array.includes`.